Only in docker-0.7.0/: build
diff -ur docker-0.7.0-o/utils/utils.go docker-0.7.0/utils/utils.go
--- docker-0.7.0-o/utils/utils.go	2013-11-26 01:09:45.000000000 -0700
+++ docker-0.7.0/utils/utils.go	2013-11-28 12:39:38.000000000 -0700
@@ -272,10 +272,6 @@
 // Figure out the path of our dockerinit (which may be SelfPath())
 func DockerInitPath(localCopy string) string {
 	selfPath := SelfPath()
-	if isValidDockerInitPath(selfPath, selfPath) {
-		// if we're valid, don't bother checking anything else
-		return selfPath
-	}
 	var possibleInits = []string{
 		localCopy,
 		filepath.Join(filepath.Dir(selfPath), "dockerinit"),
@@ -291,9 +287,7 @@
 				// LookPath already validated that this file exists and is executable (following symlinks), so how could Abs fail?
 				panic(err)
 			}
-			if isValidDockerInitPath(path, selfPath) {
-				return path
-			}
+            return path
 		}
 	}
 	return ""
Only in docker-0.7.0/utils: utils.go~
