--- a/contrib/init/systemd/docker.service
+++ b/contrib/init/systemd/docker.service
@@ -9,11 +9,13 @@
 StartLimitIntervalSec=60
 
 [Service]
+Environment=OPTIONS=
+EnvironmentFile=-/etc/sysconfig/docker
 Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
+ExecStart=/usr/sbin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock $OPTIONS
 ExecReload=/bin/kill -s HUP $MAINPID
 TimeoutStartSec=0
 RestartSec=2
